### ISSUE TITLE
Error 500 creating postage batch on 1.9.0

### ISSUE THREAD

**Issue by tmm360** [ethersphere/bee#3435]
### Context
Bee v1.9.0

### Summary
Invoked POST on a sample `/stamps/500/32` with debug api, I receive error 500.

### Expected behavior
It should work returning a postage batch id.

### Actual behavior
Console logs doesn't print any specific message:
```
"time"="2022-10-17 21:09:48.138647" "level"="info" "logger"="node/api" "msg"="debug api access" "ip"="192.168.255.6" "method"="POST" "host"="192.168.1.124:1635" "uri"="/stamps/500/32" "proto"="HTTP/1.1" "status"=500 "size"=46 "duration"="10.627506237s"
```

### Steps to reproduce
I've simply updated a previous node from 1.8.2 to 1.9.0, and executed the call.

### Possible solution
Don't know.

**Comment by ldeffenb** [comment]
If you're not running verbose logs, then you might want to crank it up and try again.   There may be more in the logs with --verbosity 5.

**Comment by tmm360** [comment]
Logs with `--verbosity 5`
[beeLogs.txt](https://github.com/ethersphere/bee/files/9804676/beeLogs.txt)
error line is 3162

**Comment by ldeffenb** [comment]
https://blockscout.com/xdai/mainnet/address/0xa5084fFebdcf630e13BAddF88531bE4d002439B3/transactions#address-tabs

**Comment by swarmHaseeb** [comment]
It might be that this was an out of gas error. 
`"time"="2022-10-17 21:29:02.520917" "level"="error" "logger"="node/api" "msg"="create batch: create failed"`
there might be a case where the tx cost is high and the tx failed because of that. A short term solution could be using custom headers but i hope with merging of https://github.com/ethersphere/bee/pull/3468 we probably will be able to avoid such issues. 
@agazso until then, if there are no further comments i think we can close this.

**Comment by agazso** [comment]
There is also another fix in order to avoid failing or stuck transactions: https://github.com/ethersphere/bee/pull/3496

**Comment by istae** [comment]
@agazso the main fix is #3486 
#3496 reduces the limit after because of the changes in #3486

**Comment by istae** [comment]
We check the balance of bzzs but not the native token (eg: xdai) so a 500 could be returned for those cases.
